incubator/gocd/templates/gocd-server-deployment.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "gocd.fullname" . }}-server
+  labels:
+    app: {{ template "gocd.name" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: {{ .Release.Name | quote }}
+    heritage: {{ .Release.Service | quote }}
+    component: server
+spec:
+  {{- if le .Values.server.replicaCount 1.0 }}
+  replicas: {{ .Values.server.replicaCount }}
+  {{- end }}
+  selector:
+    matchLabels:
+      app: {{ template "gocd.name" . }}
+      release: {{ .Release.Name | quote }}
+      component: server
+  template:
+    metadata:
+      labels:
+        app: {{ template "gocd.name" . }}
+        release: {{ .Release.Name | quote }}
+        component: server
+    spec:
+      {{- if .Values.server.persistence.enabled }}
+      volumes:
+        - name: godata-vol
+          persistentVolumeClaim:
+            claimName: {{ template "gocd.fullname" . }}-data
+        - name: homego-vol
+          persistentVolumeClaim:
+           claimName: {{ template "gocd.fullname" . }}-home
+      {{- end }}
+      containers:
+        - name: {{ template "gocd.name" . }}-server
+          {{- if .Values.server.image.tag }}
+          image: "{{ .Values.server.image.repository }}:{{ .Values.server.image.tag }}"
+          {{- else }}
+          image: "{{ .Values.server.image.repository }}:v{{ default .Chart.AppVersion .Values.app.version }}"
+          {{- end }}
+          imagePullPolicy: {{ .Values.server.image.pullPolicy }}
+          env:
+            {{- if .Values.server.env.goServerSystemProperties }}
+            - name: GO_SERVER_SYSTEM_PROPERTIES
+              value: {{ .Values.server.env.goServerSystemProperties }}
+            {{- end }}
+          ports:
+            - containerPort: 8153
+            - containerPort: 8154
+          livenessProbe:
+            httpGet:
+              path: /go/api/v1/health
+              port: 8153
+            initialDelaySeconds: {{ .Values.server.healthCheck.initialDelaySeconds }}
+            periodSeconds: {{ .Values.server.healthCheck.periodSeconds }}
+          readinessProbe:
+            httpGet:
+              path: /go/api/v1/health
+              port: 8153
+            initialDelaySeconds: {{ .Values.server.healthCheck.initialDelaySeconds }}
+          {{- if .Values.server.persistence.enabled }}
+          volumeMounts:
```

Comment:
Have you thought about using only one PVC with different subPaths? Just asking...